spec/controller/notes_controller_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "rails_helper"
+
+RSpec.describe Api::V1::Note, type: :request do
+  let!(:user) do
+    User.create!(email: "example1@gmail.com",
+                 name: "user 1",
+                 password: "123456",
+                 password_confirmation: "123456")
+  end
+  before { allow_any_instance_of(Api::V1::NotesController).to receive(:current_user) { user } }
+  describe "POST /api/v1/note" do
+    context "when create note " do
+      before(:each) do
+        post "/api/v1/notes", params: { title: "test", content: "aaaaaaaaaa" }
+      end
+      it "result success" do
+        response_json = JSON.parse response.body
+        expect(response_json["status"]).to eq(true)
```

Comment:
`        expect(response.body).to eq({
          status: true,
          data: {
            title: "test",
            content: "aaaaaa",
          }
        }.to_json)`